udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +90,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Obtains parameter from Comments typing field.
+   * Gets count for comments to be shown.
+   * Asks for 1 comment if receives non-positive input.
```

Comment:
Instead of "asks" (it's not clear what's being asked), clearer would be "returns":

```
Returns 1 if the comment limit specified in the request is 0 or negative.
```

Reply:
Will be changed in APIs PR

Reply:
Changed in APIs PR